tests/core/cxx/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
Several test comments:

1. In terms of test organization, I'd prefer this to be tests/core/cgo, since this is really testing cgo functionality. Could be divided further into language subdirectories (tests/core/cgo/cxx).
2. We really want to verify that the right set of options get passed to each language. `copts` should be C-only, `cxxopts` should be C++-only, `cppopts` should be both. Easiest way to do that is probably with `-D` flags, then checking at build time with `#if` and `#error`. This would also fix the error on Travis CI due not having a C++14 compiler available.
3. Please create a README.rst describing the tests, and run tests/update.py.